src/wannier90io/_u.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+import typing
+
+import numpy as np
+
+# TODO: implement also read_u_dis
+__all__ = ['read_u']
+
+
+def read_u(stream: typing.TextIO) -> tuple[np.ndarray, np.ndarray]:
+    """
+    Read unitary matrix file (seedname_u.mat) or the rectangular U_dis matrix
+    file (seedname_u_dis.mat).
+
+    Note: for the _u.mat file, num_bands == num_wann.
```

Comment:
```suggestion
    Note:
        for the _u.mat file, num_bands == num_wann.
```

Reply:
Having the note text on a new line gets `mkdocstrings` to render it correctly as an admonition. We might want to update the Python renderer (`mkdocstrings-python`) version, as newer versions have the admonition expanded by default (mkdocstrings/mkdocstrings#468).